app/assets/stylesheets/dashboards.scss

```diff
@@ -24,3 +24,28 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+.pt-phone,.pt-LMP {
+  width: 100px;
+  white-space: nowrap;
+  padding-right: 5px;
+}
+
+.pt-recent-notes {
```

Comment:
(actually, I think that float-right solves a diff issue, so let's keep that either way. my mistake!)